console_shortcut/menu-windows.json

```diff
@@ -0,0 +1,12 @@
+{
+    "menu_name": "Python Prompts",
+    "menu_items":
+    [
+        {
+            "name": "Conda Env Prompt",
```

Comment:
Actually, that recipe shouldn't be in this PR at all.  Thanks for pointing it out.  I'll try to clean it up.